code/modules/antagonists/eldritch_cult/knowledge/flesh_lore.dm

```diff
@@ -0,0 +1,275 @@
+
+/datum/eldritch_knowledge/flesh_ghoul/on_finished_recipe(mob/living/user,list/atoms,loc)
+	var/mob/living/carbon/human/H
+	for(var/mob/living/carbon/human/humie in atoms)
+		H = humie
+	if(!H)
+		return
+	var/mob/dead/observer/G
+	for(var/mob/dead/observer/ghost in GLOB.dead_mob_list) //we are just looking for the owner of the body
+		if(ghost.mind && ghost.mind.current == H && ghost.client)  //the dead mobs list can contain clientless mobs
+			G = ghost
+			break
+
+	if(!G || !G.reenter_corpse())
+		var/list/mob/dead/observer/candidates = pollCandidatesForMob("Do you want to play as a [H.real_name], a voiceless dead", ROLE_HERETIC, null, ROLE_HERETIC, 50,H)
+		if(!LAZYLEN(candidates))
+			return
+		var/mob/dead/observer/C = pick(candidates)
+		message_admins("[key_name_admin(C)] has taken control of ([key_name_admin(H)]) to replace an AFK player.")
+		H.ghostize(0)
+		H.key = C.key
+
+	if(!H.mind || !H.client)
+		to_chat(user, "<span class='warning'>There is no soul connected to this body...</span>")
+		return
+
+	if(!check_ghouls(user) || HAS_TRAIT(H,TRAIT_HUSK))
+		return
+
+	ADD_TRAIT(H,TRAIT_MUTE,MAGIC_TRAIT)
+
+	H.revive(full_heal = TRUE, admin_revive = TRUE)
+	H.setMaxHealth(50)
+	H.health = 50 // Voiceless dead are much tougher than ghouls
+	H.become_husk()
+	H.faction |= "e_cult"
+	H.fully_replace_character_name(H.real_name,"Voiceless [H.real_name]")
+	to_chat(H, "<span class='userdanger'>You have been revived by </span><B>[user.real_name]!</B>")
+	to_chat(H, "<span class='userdanger'>[user.p_theyre(TRUE)] your master now, assist [user.p_them()] even if it costs you your new life!</span>")
+	atoms -= H
+	ghouls += H
+
+/datum/eldritch_knowledge/flesh_ghoul/proc/check_ghouls(mob/living/user)
+
+	listclearnulls(ghouls)
+
+	for(var/mob/living/carbon/human/ghoul in ghouls)
+		if(ghoul.stat == DEAD)
+			to_chat(user, "<span class='big bold'>You feel the evil influence leave your body... you are no longer enslaved to [user.real_name]</span>")
+			ghouls -= ghoul
+			current_amt--
+
+	if(current_amt >= max_amt)
+		return FALSE
+	return TRUE
+
+/datum/eldritch_knowledge/flesh_grasp
+	name = "Grasp of Flesh"
+	gain_text = "My new found desire, it drove me to do great things! The Priest said."
+	desc = "Empowers your mansus grasp to be able to create a single ghoul out of a dead person. Ghouls have only 25 HP and look like husks."
+	cost = 1
+	next_knowledge = list(/datum/eldritch_knowledge/flesh_ghoul)
+	var/ghoul_amt = 1
+	var/list/spooky_scaries
+	route = "Flesh"
+
+/datum/eldritch_knowledge/flesh_grasp/on_mansus_grasp(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(!ishumanbasic(target))
+		return
+	var/mob/living/carbon/human/H = target
+	var/datum/status_effect/eldritch/E = H.has_status_effect(/datum/status_effect/eldritch/rust) || H.has_status_effect(/datum/status_effect/eldritch/ash) || H.has_status_effect(/datum/status_effect/eldritch/flesh)
+	if(E)
+		E.on_effect()
+		H.bleed_rate = min(H.bleed_rate + 4, 8)
```

Comment:
Marks triggering on grasps sounds good on paper but two of the three grasps don't really do anything when targeting normal humans. This leaves ash to be the only path who can really take full advantage of that mechanic.

I feel like this ability and Imperfect ritual step on each others toes anyhow. Both revive slightly weaker humans which is kind of weird. Maybe replace it with something more offensively oriented to make the marks more useful?

Reply:
the mark doesnt resurrect em, the flesh grasp does. And the difrrence between the standard ghouls is that you can have only 1 and they have 25 hp. Voiceless dead have 50hp, you can have 2 at a time and are mute. The mechanics are quite simmiliar, but the whole point of flesh is that you get to build your basically necromancer army

Reply:
but yeah i have to add more unique effects